stage #1/module #1. Intro, Infrastructure & Instruments, Fundamentals/lesson #3. Classes, Intro to OOP/Weapon/Weapon.iml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4" />
```

Comment:
do not commit IDE files, commit source code only

Reply:
Done